py/fastspecfit/io.py

```diff
@@ -1368,6 +1368,14 @@ def read_stacked(self, stackfiles, firsttarget=0, ntargets=None,
             
             ivar = fitsio.read(stackfile, 'IVAR')
             ivar = ivar[fitindx, :]
+            
+            # Check if the file contains a resolution matrix, if it does not
+            # then use an identity matrix
+            try:
+                res = fitsio.read(stackfile, 'RES')
+                res = res[fitindx, :, :]
+            except:
+                res = np.stack([np.ones((1, npix)) for i in range(nobj)], axis=0) # Hack!
 
```

Comment:
Rather than a `try`:`except` block can you please explicitly check for the extension. I think something like
```
F = fitsio.FITS(stackfile)
if 'REF' in F:
    res = F['REF'].read()
```

Reply:
Removed the try: except.